scripts/ci/6-check-license.sh

```diff
@@ -17,13 +16,29 @@
 # limitations under the License.
 #
 
+declare -r RAT_VERSION=0.12
+declare -r URL="http://repo1.maven.org/maven2/org/apache/rat/apache-rat/${RAT_VERSION}/apache-rat-${RAT_VERSION}.jar"
 
-acquire_rat_jar () {
+declare -r TMP_DIR=/tmp
+declare -r rat_jar="${TMP_DIR}"/lib/apache-rat-${RAT_VERSION}.jar
```

Comment:
I believe double quote after TMP_DIR var should be at the end of the line.

Reply:
It's perhaps more conventional there but this works fine

```
$ echo "$HOME"
/Users/ash
$ echo "$HOME"def
/Users/ashdef
```

Reply:
Yes, I'm aware it works, but looking at the fact that there is furhter on `${RAT_VERSION}` I would rather expect the whole `declare -r rat_jar=` to be double quoted.